README.md

```diff
@@ -286,6 +305,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 *Libraries for network programming.*
 
 * [Grizzly](https://grizzly.java.net/) - NIO framework. Used as a network layer in Glassfish.
+* [Jodd HTTP](http://jodd.org/doc/http.html) - developer-friendly fluent HTTP library.
```

Comment:
The naming is really bad. HTTP, EMail, JSON. Only discernible by identifying it at the same time with Jodd.